animata/card/scorecard.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from "react";
+import { motion } from "framer-motion";
+interface Team {
+  name: string;
+  logo: string;
+}
+
+interface ScoreProps {
+  homeTeam: Team;
+  awayTeam: Team;
+  homeScore: number;
+  awayScore: number;
+  matchTime: string;
+  scorer: string;
+}
+
+export default function LiveScoreWidget({
+  homeTeam,
+  awayTeam,
+  homeScore,
+  awayScore,
+  matchTime,
+  scorer,
+}: ScoreProps) {
+  const [y, setY] = useState(28);
+  const [hmScore] = useState(homeScore);
+  const [awScore, setAwScore] = useState(awayScore);
+  const [scored, setScored] = useState(false);
+  return (
+    <div className="p-10">
+      <div className={`rounded-3xl ${scored ? "bg-[#09374d]" : "bg-none"} p-1`}>
+        <motion.div className="box" animate={{ y }} transition={{ type: "spring" }}>
+          <div className="mx-auto flex w-[440px] items-center justify-evenly rounded-3xl bg-black p-4 text-white">
+            {/* Home Team h-[135px] */}
+            <div className="flex flex-col">
+              <img
+                src={homeTeam.logo}
+                alt={homeTeam.name}
+                className="mb-2 h-[70px] w-[70px] object-contain"
+              />
+              {!scored && <span>{homeTeam.name}</span>}
+            </div>
+
+            {/* Score */}
+            <div className="text-center">
+              <div className="text-4xl font-bold">
+                {hmScore}-{awScore}
+              </div>
+              <div className="text-base">{matchTime}</div>
+            </div>
+
+            {/* Away Team */}
+            <div className="flex flex-col">
+              <img
+                src={awayTeam.logo}
+                alt={awayTeam.name}
+                className="mb-2 h-[70px] w-[70px] object-contain"
+              />
+              {!scored && <span>{awayTeam.name}</span>}
+            </div>
+          </div>
+        </motion.div>
+        {scored && (
+          <div className="flex justify-between py-4">
+            <div className="ml-2 flex items-center text-lg text-white">
+              <FootbalIcon />
+              <span className="ml-2 text-white">{scorer} scores!</span>
+            </div>
+            <span className="mr-2 text-lg text-slate-500">now</span>
+          </div>
+        )}
+      </div>
```

Comment:
You can animate the height property to make the animation feel smooth. Given we have a fixed height, you can animate it from 0 to the fixed value and the animation will look better. You probably will not need the `y` animation then.